tmcprototype/SubarrayNode/SubarrayNode/SubarrayNode.py

```diff
@@ -418,14 +418,17 @@ def Configure(self, argin):
         :return: None
         """
         try:
-            for i in range(0, len(argin)):
-                if type(float(argin[i])) == float:
-                    pass
+            # Validation to check float argins
```

Comment:
Remove commented code if not required.

Reply:
Done.

Reply:
Commit id is: [01a44358029a01fcc943c419fc4d7606eab518be](https://github.com/ska-telescope/tmc-prototype/commit/01a44358029a01fcc943c419fc4d7606eab518be)